lib/validator/configValidator.py

```diff
@@ -125,6 +125,24 @@ def get_config_and_schema(parsed_folder):
         return config_ret, schema_ret
 
     @staticmethod
+    def get_mapping_config_and_schema(input_folder):
```

Comment:
please adding some description comments for describing the input and output of this method.